elastic/elastic-agent
# [chore][otel] Enable persistence in otel.yml

- Enhancement
- Docs

## What does this PR do?

This PR adds the `filestorage` extension in `otel.yaml` and updates the documentation.
Updates `otel` dependencies to 0.110.0

## Why is it important?

By default, the OpenTelemetry Collector operates in a stateless mode, meaning it does not save file offsets to disk. To resolve this, we enable persistence in the default configuration files.

## Checklist


## How to test this PR locally

- Build from this branch
- Keep the config default and run `elastic-agent otel --config otel.yaml`
- Ingest some data and stop the process.
- Restart the process and ingest some more data.
- You shouldn't see any data loss/duplication.
## Related issues

- Closes https://github.com/elastic/opentelemetry-dev/issues/281
